### DISCUSSION TITLE
已支付一年的费用，但并未升级专业版

### DISCUSSION THREAD

**Discussion by hhansen663** [getlantern/lantern-forum-cn#1021] [Q&A]
![Uploading 微信图片_20241104082859.jpg…]()
361003297@qq.com

**Comment by hhansen663** [comment]
![Uploading 微信图片_20241104082859.jpg…]()

**Comment by hhansen663** [comment]
![Uploading 微信图片_20241104082859.jpg…]()

**Comment by hhansen663** [comment]
![Uploading 微信图片_20241104082859.jpg…]()

**Comment by hhansen663** [comment]
![微信图片_20241104082859](https://github.com/user-attachments/assets/5413538c-3229-4836-a63e-c37884ed8e3f)

**Comment by hhansen663** [comment]
361003297@qq.com

**Comment by MaxMKMax** [comment]
@hhansen663 您好. 已经帮您提交问题上去了. 谢谢您的耐心等待.

**Reply by hhansen663** [reply]
大概要等多久呢

**Reply by MaxMKMax** [reply]
@hhansen663 这是您的专属激活码:

DD7J4-6T964-4JQ3B-6QC4V-7RQKH
